org.csstudio.display.builder.util/src/org/csstudio/display/builder/util/ResourceUtil.java

```diff
@@ -45,7 +49,12 @@ public static InputStream openPlatformResource(final String resource_name) throw
             // Handle "platform://.." path during tests in the 'main' directory of a plugin,
             // so "../" leads to the parent of all plugin sources, from which we
             // then locate "specific_plugin/dir/file.png"
-            final String resolved = resource_name.replace("platform:/plugin/", "../");
+            String resolved = resource_name;
+
+            if ( resource_name.contains("platform:/") ) {
+                resolved = URIUtil.toFile(new URI(FileLocator.resolve(new URL(resource_name)).toExternalForm())).getCanonicalPath();
+            }
+
```

Comment:
The change here allows any `platform:/` URI to be used.
It works well for my test where I use `platform:/config/...`.
Can you verify that nothing of your is broken?